src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -293,6 +277,47 @@ public bool BringDescendantIntoView(Visual target, Rect targetRect)
             return !Offset.NearlyEquals(oldOffset);
         }
 
+        /// <summary>
+        /// Computes the closest offset to ensure most of the child is visible in the viewport along an axis.
+        /// </summary>
+        /// <param name="viewportStart">The left or top of the viewport</param>
+        /// <param name="viewportEnd">The right or bottom of the viewport</param>
+        /// <param name="childStart">The left or top of the child</param>
+        /// <param name="childEnd">The right or bottom of the child</param>
+        /// <returns></returns>
+        internal static double ComputeScrollOffsetWithMinimalScroll(
+            double viewportStart,
+            double viewportEnd,
+            double childStart,
+            double childEnd)
+        {
+            // If child is above viewport, i.e. top of child is above viewport top and bottom of child is above viewport bottom.
```

Comment:
Minor, but this comment should read _is at least partially above the viewport_, or something along those lines.
The variable naming (`isChildAbove`) initially made me think that this was checking whether the child was _entirely_ outside the viewport, and the comment reinforced that idea.

(Same thing for `isChildBelow`)

Reply:
Updated.